Content.Shared/Clothing/EntitySystems/ClothingSystem.cs

```diff
@@ -98,6 +98,18 @@ private void ToggleVisualLayers(EntityUid equipee, HashSet<HumanoidVisualLayers>
                 {
                     if (comp.Slots.Contains(layer))
                     {
+                        //ss220-ToggleVisualLayers-Fix begin
+                        //FunTust:Check where the item is placed and if it is not one of the clothing slots then do not touch the visibility
+                        _invSystem.TryGetContainingSlot(item, out var currentItemSlot);
+                        if(currentItemSlot != null)
+                        {
+                            if (currentItemSlot.SlotGroup != "Default")
+                            {
+                                break;
+                            }
+                        }
```

Comment:
Во избежание спагетти кода рекомендую сделать так:
```suggestion
                        if (_invSystem.TryGetContainingSlot(item, out var currentItemSlot)
                        && currentItemSlot.SlotGroup != "Default")
                                break;
```
т. к. `TryGetContainingSlot()` возвращает true, если `currentItemSlot != null`